channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1132,14 +1157,15 @@ UINT rdpdr_send(rdpdrPlugin* rdpdr, wStream* s)
 	UINT status;
 	rdpdrPlugin* plugin = (rdpdrPlugin*) rdpdr;
 
+	if (!rdpdr || !s)
```

Comment:
@hardening The function is not private to the file, so I'd assume such things can happen (but you're right, not that much of a threat).